puppet/modules/jenkins_job_builder/files/config/scripts/packaging_repoclosure.sh

```diff
@@ -41,5 +41,15 @@ if [ -n "${copr_lookasides}" ]; then
   done
 fi
 
+puppet_lookaside=""
+if [ -n "${puppet_lookasides}" ]; then
+  lookaside_repos=$(echo $puppet_lookasides | tr "," "\n")
+
+  for lookaside_repo in $lookaside_repos
+  do
+    puppet_lookaside+="--repofrompath=${lookaside_repo},http://yum.puppetlabs.com/el/${osver}/products/x86_64/ -l ${lookaside_repo} "
```

Comment:
Could you perhaps put the el/ inside the string?  We use this repoclosure script for Fedora too, so would be good to have the option to use that repo.